Classes/NumberTest.cpp

```diff
@@ -255,9 +257,13 @@ void NumberTest::update(bool hit)
 
 		auto * enableMenuLambda = CallFunc::create([this, menu]() {
 			menu->setEnabled(true);
-			addTut(Vec2(visibleSize.width - 100, visibleSize.height / 2 - 230));
-			addTut(Vec2(visibleSize.width - 100, visibleSize.height / 2  - 60));
-			addTut(Vec2(visibleSize.width - 100, visibleSize.height / 2 + 120));
+			if (NumberTest::tutActive)
+			{
+				addTut(Vec2(visibleSize.width - 100, visibleSize.height / 2 - 170));
```

Comment:
@protito What does this code do? Can you add some comments explaining what these calculations are for?